concepticondata/conceptlists/Wittmann-1973-200.tsv

```diff
@@ -0,0 +1,201 @@
+ID	NUMBER	CONCEPTICON_ID	CONCEPTICON_GLOSS	ENGLISH	MAURITIAN	HAITIAN	FRENCH
+Wittmann-1973-200-1	1	98	ALL	all	tu	tu	tout
+Wittmann-1973-200-2	2	1577	AND	and	ek	é, épi	et
+Wittmann-1973-200-3	3	619	ANIMAL	animal	zanimo	bèt	animal
+Wittmann-1973-200-4	4	646	ASH	ashes	lasan	sân	cendres
+Wittmann-1973-200-5	5	1461	AT	at		a	à
```

Comment:
Is the MAURITIAN form missing in the original data set as well?

Reply:
In the original article, the missing form is represented as "Ø". I left it blank here so it's not interpreted as the corresponding letter. Should I rather change it to Ø?

Reply:
Ø usually refers to a 'null-morpheme'. You could add "Ø (null-morpheme)" in the cell so that it doesn't look like a letter or missing data point. @LinguList what do you think?

Reply:
It is difficult to say what Ø refers to in lexicostatistical datasets, so I'd rather also leave it blank. What one can also do, since this is supposed to be a lexical dataset, is to delete the language and leave only those which serve as glossing languages? In fact, even the French as a lexifier is potentially something we may not need, since they obviously chose some "translations" in such a way that the words are cognate!

Reply:
Maybe someone else wants to chime in to decide this? I would leave all four languages in there, especially French, because while the list is based on the English Swadesh concepts, the description sounded very much like French was the point of departure. It also makes some non-optimal translations of Swadesh's original more visible if Haitian and Mauritian are represented.